addOns/requester/src/main/java/org/zaproxy/addon/requester/internal/ManualHttpRequestEditorPanel.java

```diff
@@ -56,7 +58,7 @@
 import org.zaproxy.zap.view.HttpPanelManager;
 
 @SuppressWarnings("serial")
-public class ManualHttpRequestEditorPanel extends ManualRequestEditorPanel
+public class ManualHttpRequestEditorPanel extends MessageEditorPanel
```

Comment:
`ManualHttpMessageEditorPanel`?

Reply:
Happy to change it, but it only allows to edit the request.